rplugin/python3/deoplete/sources/beancount.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, vim):
         self.name = 'beancount'
         self.mark = '[bc]'
         self.filetypes = ['beancount']
+        self.rank = 500
```

Comment:
I found I had to do something else because the buffer and around sources were so much slower than the beancount one.  It's unclear whether or not the other sources are running for me, but I'll figure it out.